tools/misc/get_plz.sh

```diff
@@ -38,7 +38,7 @@ echo "Downloading Please ${VERSION}..."
 curl -fsSL "${PLEASE_URL}" | tar -xzpf- --strip-components=1 -C "$DIR"
 # Link it all back up a dir
 for x in "${DIR}/"*; do
```

Comment:
Just trying to figure out why that PR broke this. Am I right in thinking that `ls $DIR/*` gives "please", whereas "$DIR/*" gives "$DIR/foo"?